home/home2.2/image.java

```diff
@@ -0,0 +1,184 @@
+import java.io.FileWriter;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.BufferedWriter;
+
+class Constants {
+    public static final int MatrixSize = 16;
+    public static final int Fine = 0;
+    public static final int sizeProblem = 1;
+    public static final int colorProblem = 2;
+}
+
+class Color {
+    int r = -1;
+    int g = -1;
+    int b = -1;
+}
+
+class Pixel{
+    
+    public Pixel() {}
+
+    public Pixel(String col) {
+        String[] values = col.split(",");
+        if (values.length == 3) {
+            color.r = convertToInt(values[0]);
+            color.g = convertToInt(values[1]);
+            color.b = convertToInt(values[2]);
+        }
+    }
+
+    public Color getColor() {
+        return color;
+    }
+
+    public void setColor(Pixel col){
+        this.color=col.getColor();
+    }
+    
+    public String toString() {
+        return color.r + "," + color.g + "," + color.b;
```

Comment:
String::format can be used for such tasks. It can be more efficient then + as + uses chain of appends and string copies. format should be the fastest way to do the trick